WatcherBot/Config/Config.cs

```diff
@@ -9,7 +9,7 @@ namespace WatcherBot.Config;
 public class Config
 {
     public const string ConfigSection = "Watcher";
-    private readonly Lazy<Dictionary<ulong, Range>> attachmentLimitsLazy;
+    private readonly System.Lazy<Dictionary<ulong, Range>> attachmentLimitsLazy;
```

Comment:
Why `System.`?

Reply:
I don't know where this change originated from. I blame my IDE.